bench-tps/src/cli.rs

```diff
@@ -513,5 +534,55 @@ pub fn extract_args(matches: &ArgMatches) -> Config {
         );
     }
 
+    if let Some(addr) = matches.value_of("bind_address") {
+        args.bind_address = solana_net_utils::parse_host(addr).unwrap_or_else(|e| {
+            eprintln!("failed to parse bind_address address: {e}");
+            exit(1)
+        });
+    }
+    let (_, node_id_path) = ConfigInput::compute_keypair_path_setting(
+        matches.value_of("client_node_id").unwrap_or(""),
+        &config.keypair_path,
+    );
+    if let Ok(node_id) = read_keypair_file(node_id_path) {
+        args.client_node_id = Some(node_id);
+    } else if matches.is_present("client_node_id") {
+        panic!("could not parse identity path");
```

Comment:
-> Could not parse identity path

Can you use match and actually print the real error returned from read_keypair_file as there can be multiple reasons for the errors like permission, file not found , not protected and etc.?

Reply:
I think I can use clap validator for this case. I want to rewrite this cli parsing code in the follow up PR completely.